src/main/service/LocationService.ts

```diff
@@ -99,4 +99,14 @@ export class LocationService {
     public async deleteLocationById(locationId: number, requester: string): Promise<object> {
         return await locationRequest.deleteCourt(locationId, requester);
     }
+
+    public async findCourtsJurisdiction(locations): Promise<object> {
```

Comment:
Can we change the returned type to Promise<string[]> and not object?

Reply:
done, thanks, good catch :)